components/AssetFormTitle/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image';
+import {
+  TitleContainer,
+  Name,
+  General,
+  CollectionName,
+  CollectionCreator,
+  CollectionIconContainer,
+} from './AssetFormTitle.styled';
+
+type Props = {
+  name: string;
+};
+
+const AssetFormTitle = ({ name }: Props): JSX.Element => {
+  return (
+    <>
+      <CollectionIconContainer>
+        <Image
+          priority
+          layout="fixed"
+          width={24}
+          height={24}
+          src="/icon-monsters.png"
+          alt="Crypto Monsters icon"
+        />
+        <CollectionName>Crypto Monsters</CollectionName>
```

Comment:
Also I don't see the icon-monsters.png, it's not showing up for me. This should actually pull the name of the collection for this template. Capitalize the first letter of course. Font here needs to also match mocks.

Reply:
Oh oops - both of these will show up on the Marketplace details page and My NFTs details page when reviewing WEB-833-my-template-detail. I'll leave this comment as unresolved to double check later when the other two PRs are reviewed/merged!